libraries/AP_Notify/Display.cpp

```diff
@@ -449,6 +449,7 @@ void Display::draw_char(uint16_t x, uint16_t y, const char c)
 void Display::update_arm(uint8_t r)
 {
     if (AP_Notify::flags.armed) {
+        draw_text(COLUMN(0), ROW(r-1), ">>> AgroRaptor <<<");
```

Comment:
Mistake? This doesn't look to be generally ArduPilot related, but is instead tied to a specific vehicle.